lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -100,7 +100,7 @@
                              el: '.js-course-list-completed',
                              childView: CourseCardView,
                              collection: this.completedCourseCollection,
-                             context: this.options
+                             context: $.extend(this.options, {collection_status: 'completed'})
```

Comment:
If we wanted constants for `'completed'`, is there a reasonable place to implement this?

Reply:
I don't think there's a good place for this given how things get handled with RequireJS and with the Underscore templates, unfortunately.